<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneviewerEditorWidget</class>
 <widget class="QWidget" name="SceneviewerEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sceneviewer Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="view_all_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>View All</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="antialias_label">
     <property name="text">
      <string>Antialias:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="antialias">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="perspective_checkbox">
     <property name="text">
      <string>Perspective projection</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="view_angle_label">
     <property name="text">
      <string>View angle:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="view_angle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="eye_position_label">
     <property name="text">
      <string>Eye position:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="eye_position">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lookat_position_label">
     <property name="text">
      <string>Look at position:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lookat_position">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="up_vector_label">
     <property name="text">
      <string>Up vector:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="up_vector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="clipping_planes_groupbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Clipping planes:</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="near_clipping_label">
        <property name="text">
         <string>Near:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="far_clipping_label">
        <property name="text">
         <string>Far:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="near_clipping_slider">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="far_clipping_slider">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="background_colour_label">
     <property name="text">
      <string>Background colour R, G, B:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="background_colour">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="light_both_sides_checkbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Light Both Sides</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="perturbline_checkbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Perturb Line</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>view_all_button</sender>
   <signal>clicked()</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>viewAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perspective_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>perspectiveStateChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view_angle</sender>
   <signal>editingFinished()</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>viewAngleEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eye_position</sender>
   <signal>editingFinished()</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>eyePositionEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lookat_position</sender>
   <signal>editingFinished()</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>lookatPositionEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>up_vector</sender>
   <signal>editingFinished()</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>upVectorEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>near_clipping_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>nearClippingChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>far_clipping_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>farClippingChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>background_colour</sender>
   <signal>editingFinished()</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>backgroundColourEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>antialias</sender>
   <signal>editingFinished()</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>antialiasEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perturbline_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>perturbLineStateChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>light_both_sides_checkbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>SceneviewerEditorWidget</receiver>
   <slot>lightBothSidesStateChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>viewAll()</slot>
  <slot>viewAngleEntered()</slot>
  <slot>lookatPositionEntered()</slot>
  <slot>eyePositionEntered()</slot>
  <slot>upVectorEntered()</slot>
  <slot>perspectiveStateChanged(bool)</slot>
  <slot>nearClippingChanged(int)</slot>
  <slot>farClippingChanged(int)</slot>
  <slot>backgroundColourEntered()</slot>
  <slot>antialiasEntered()</slot>
  <slot>lightBothsidesStateChanged(bool)</slot>
  <slot>perturbLineStateChanged(bool)</slot>
  <slot>lightBothSidesStateChanged(bool)</slot>
 </slots>
</ui>
